File: Python/AdivinhaNumeroGame.py
import random

class AdivinhaNumeroGame:
    numeroSorteado = random.randint(1,100)
    print('"Bem vindo ao jogo da adivinhação')
    chute = 0
    tentativas = 0

    while chute != numeroSorteado:
        chute = int(input('Qual o número você acha que o computador pensou? '))
        tentativas += 1
        if chute > numeroSorteado:
            print('O número é menor')
        elif chute < numeroSorteado:
            print('O número é maior')
    
    palavraTentativas = ' tentativa' if tentativas == 1 else ' tentativas'
    print(f'Parabéns você acertou com {tentativas} {palavraTentativas}')

    

File: Java/AdivinhaNumeroGame.java
 package Java;

import java.util.Scanner;

class AdivinhaNumeroGame {
    public static void main(String[] args) {
        System.out.println("Bem vindo ao jogo da adivinhação");
        int numeroSorteio = 10;

        Scanner sc = new Scanner(System.in);
        System.out.println("Qual o número você acha que o computador pensou? ");
        int chute = sc.nextInt();

        if (chute == numeroSorteio) {
            System.out.println("Numero correto");
        }
        else{
            System.out.println("Infelizmente você errou");
        }

        sc.close();

    }
    
}